libcudacxx/test/libcudacxx/cuda/annotated_ptr/utils.h

```diff
@@ -8,7 +8,7 @@
 //
 //===----------------------------------------------------------------------===//
 
-#if defined(_LIBCUDACXX_COMPILER_MSVC)
```

Comment:
This requires an include of "test_macros.h" for the macro to be defined

Reply:
added include for "test_macros.h"